### PR TITLE
swm: init at 2015-07-07

### PR DESCRIPTION
**Pull request by franksn**
wmutils-opt: init at 2015-08-01

### CODE REVIEW

File: pkgs/applications/window-managers/swm/default.nix [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    '';
+
+    installPhase = ''
+      make PREFIX=$out install
+    '';
+
+    meta = {
+      description = "Simple Window Manager";
+      homepage = http://github.com/dcat/swm;
+      maintainers = [ "Alex Frank Adhyatma <franksn@openmailbox.org>" ];
```
Comment (pSub) [inline pkgs/applications/window-managers/swm/default.nix]: Please add yourself to `lib/maintainers.nix` and use the identifier here.

File: pkgs/tools/X11/wmutils-opt/default.nix [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    rev = "00fb88f80f2c42cdd664dc678430e77587cd392c";
+    sha256 = "0938dnx9ql0b91igw9j59grfcjhgn7s31pdvb1ixfs6w4d2g1kcr";
+  };
+
+  buildInputs = [ libxcb ];
+
+  installFlags = [ "PREFIX=$(out)" ];
+
+  meta = with stdenv.lib; {
+    description = "optional addons to wmutils";
```
Comment (pSub) [inline pkgs/tools/X11/wmutils-opt/default.nix]: `description` should start with a captial letter.

**Comment by franksn** [comment]
edited.

File: nixos/modules/services/x11/window-managers/swm.nix [hunk 3]
```diff
@@ -0,0 +1,28 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.xserver.windowManager.swm;
+in
+{
+  options = {
+    services.xserver.windowManager.swm.enable = mkOption {
```
Comment (jagajaga) [inline nixos/modules/services/x11/window-managers/swm.nix]: Please use `mkEnableOption`/

Comment (franksn) [inline nixos/modules/services/x11/window-managers/swm.nix]: done sir!

**Comment by matthiasbeyer** [comment]
Why are there two commits with the same commit message but different contents? I guess this should be fixed before merging.

**Comment by franksn** [comment]
I'm really sorry, i haven't thoroughly read the contributor's guide before as this is my first PR here. Sorry for the inconvenience guys.

**Pull request closed without merging**